system-tests/projects/runner-e2e-specs/cypress/e2e/errors/unexpected.cy.js

```diff
@@ -13,7 +13,7 @@ describe('unexpected errors', { defaultCommandTimeout: 0 }, () => {
   })
 
   it('Cypress method error', () => {
-    window.eval('Cypress.LocalStorage._isSpecialKeyword = () => { throw new Error("thrown in Cypress-LocalStorage-_isSpecialKeyword") }')
+    window.eval('Cypress.LocalStorage.clear = () => { throw new Error("thrown in Cypress-LocalStorage-clear") }')
```

Comment:
Was this a TODO?

I don't understand the need for this change in this PR. 

It is fine if the need is clear. I just don't know what I am reviewing here.

Reply:
Ah yes, this test is run as part of https://github.com/cypress-io/cypress/pull/20612/files#r826335410. So this change actually causes the test to fail.